minio/api.py

```diff
@@ -389,6 +389,7 @@ def set_bucket_policy(self, bucket_name, policy):
         :param bucket_name: Bucket name.
         :param policy: Access policy/ies in JSON format.
         """
+        is_valid_bucket_type_policy(policy)
```

Comment:
is_valid_bucket_policy_type would be better

Reply:
what about is_valid_policy_type as @ebozduman suggested